src/js/custom-modal.js

```diff
@@ -0,0 +1,24 @@
+// Custom Modal toggle script for when we need to use something homegrown
```

Comment:
The slideout in dev was updated to utilize the collapse/card components instead of modals, so the custom-modal data toggle isn't being used anymore.  Do we still need to add this to the theme?

Reply:
Nope. Forgot we made that change. This also takes care of @rjucf 's concern.